src/apicast-cli/cli/busted.lua

```diff
@@ -0,0 +1,16 @@
+local actions = require "apicast-cli.cmd"
+local colors = require "ansicolors"
+
+local function call(_, parser)
+  local busted_run = parser:command("b busted", "Runs busted tests.")
+    :action(actions.busted.run)
+
+  busted_run:option("-p", "only run test files matching the Lua pattern")
```

Comment:
I see. We really don't have to parse those options here. Maybe there is some setting to ignore unknown ones?